stock_location_orderpoint/models/stock_location_orderpoint.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from copy import copy
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv import expression
+from odoo.tools import float_compare, groupby, split_every
+
+
+class StockLocationOrderpoint(models.Model):
+    _name = "stock.location.orderpoint"
+    _description = "Stock location orderpoint"
+    _order = "sequence"
+
+    name = fields.Char(
+        "Name",
+        copy=False,
+        required=True,
+        readonly=True,
+        default=lambda self: self.env["ir.sequence"].next_by_code(
+            "stock.location.orderpoint"
+        ),
+    )
+    company_id = fields.Many2one(
+        "res.company",
+        "Company",
+        required=True,
+        default=lambda self: self.env.company,
+    )
+    location_id = fields.Many2one(
+        "stock.location",
+        "Location",
+        ondelete="cascade",
+        required=True,
+        check_company=True,
+        domain='[("company_id", "=", company_id)]',
+    )
+    trigger = fields.Selection(
+        [("auto", "Auto"), ("manual", "Manual")],
```

Comment:
I would maybe rename auto into scheduled (cron) and use auto/realtime based on new moves triggers.
I would generate the cron linked to this record and allow to define the periodicity on this model.

Reply:
Yes cron we be better. i will do this.
But we can not implement one cron for one orderpoint.
This must be a computed cron because we can have multiple orderpoints for one location.

Reply:
indeed, let's keep it like you did